proselint/tools.py

```diff
@@ -192,10 +184,12 @@ def load_options(config_file_path=None):
         if os.path.isfile(path):
             user_options = json.load(open(path))
             break
+        if os.path.isfile(path.replace(".json", "")):
```

Comment:
I might not be following what this is doing. Will this raise an exception if someone has a .proselintrc file but no .proselintrc.json file? It so, it might be better to give a deprecation warning using `warnings.warn` (https://docs.python.org/3/library/warnings.html#warnings.warn) and still process the .proselintrc file.

Reply:
Honestly, I wasn't aware of the existence of a `warnings` library - I have yet to deal with a package deprecation in Python, largely. Always fun to learn new things about a language even over 7 years after starting to use it :)